view/adminhtml/templates/order/view/tab/taxjar/info.phtml

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Taxjar_SalesTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Taxjar
+ * @package    Taxjar_SalesTax
+ * @copyright  Copyright (c) 2017 TaxJar. TaxJar is a trademark of TPS Unlimited, Inc. (http://www.taxjar.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+/** @var $block \Taxjar\SalesTax\Block\Adminhtml\Order\View\Tab\Taxjar\View\Info */
+
+$order = $block->getOrder();
+
+$orderAdminDate = $block->formatDate(
+    $block->getOrderSyncedAtDate($order->getData('tj_salestax_sync_date')),
+    \IntlDateFormatter::MEDIUM,
+    true
+);
+
+?>
+
+<section class="admin__page-section">
+    <div class="admin__page-section-title">
+        <span class="title"><?= $block->escapeHtml(__('Taxjar Information')) ?></span>
```

Comment:
Should the title be 'TaxJar Information' rather than 'Taxjar Information'?

Reply:
YES!
Good 👀!